Makefile

```diff
@@ -21,6 +21,7 @@ build-base: build/Dockerfile
 # TODO: Change the render to allow for the permissions to have a list of all the webhook names
 # TODO: Pull that list of names from the yaml files?
 render: $(TEMPLATEFILES) build/Dockerfile
+	mkdir deploy || true ;\
```

Comment:
This will spuriously pass if `mkdir deploy` fails for reasons other than the directory already existing (e.g. `EPERM`, or there's a regular file called `deploy` in the way). Better to use `mkdir -p deploy` and no `|| true`, since you presumably want to abort if one of those other things is wrong. (Um, I'm only guessing make targets implicitly run with `-e`?)

Reply:
@2uasimojo That's a really good point. I think this PR is pretty stale, so it can be closed. I'll certainly keep an eye on this sort of thing.